### ISSUE TITLE
关于AGDA的问题

### ISSUE THREAD

**Issue by phinoo** [yoctta/multiple-attention#22]
您好，大家都是中国人，我就用中文表达了（/尴尬，怕英文表达不准确）
在AGDA.py中有一个 mod_func的函数，最后两行代码不知道代表啥数学意义，有看明白的大佬吗？
bottom=torch.sigmoid((torch.tensor(0.)-thres)*zoom)
return (torch.sigmoid((x-thres)*zoom)-bottom)/(1-bottom)

**Comment by wangjbo** [comment]
hxd，你这个数据集有吗？可以分享一下吗 708873100@qq.com 跪求

**Comment by liruotong123** [comment]
> hxd，你这个数据集有吗？可以分享一下吗 [708873100@qq.com](mailto:708873100@qq.com) 跪求

你拿到数据集了吗，如果拿到了可以分享给我一份吗liruotong99202@163.com 球球了

**Comment by daybrak** [comment]
不好意思打扰以下，请问你有跑通这个代码吗？

**Comment by daybrak** [comment]
不好意思打扰一下，请问你有跑通这个代码吗？
